### ISSUE TITLE
Debug simulation crashes

### ISSUE THREAD

**Issue by bas-rustenburg** [choderalab/constph-openmm#45]
As a reminder to do some debugging, I'm logging some input on simulations that failed, in case the issues are related to the code.

Failed simulations that will need debugging. 
- ./ALK-Alectinib/3AOX_fixed_ph7.4.pdb.ligand/failed.txt
- ./VEGFR1-Axitinib/4AG8_fixed_ph7.4.pdb.ligand/failed.txt
- ./ALK-Crizotinib/2XP2_fixed_ph7.4.pdb.ligand/failed.txt
- ./ALK-Crizotinib/2XP2_fixed_ph7.4.pdb.apo/failed.txt
- ./MET-Crizotinib/2WGJ_fixed_ph7.4.pdb.ligand/failed.txt
- ./BCRABL-Dasatinib/2GQG_fixed_ph7.4.pdb.ligand/failed.txt
- ./VEGFR1-Regorafenib/2QU5_fixed_ph7.4.pdb.ligand/failed.txt

List of all simulations, failed ones in **bold**

- **./ALK-Alectinib/3AOX_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./ALK-Alectinib/3AOX_fixed_ph7.4.pdb.apo/trajectory.dcd
- **./VEGFR1-Axitinib/4AG8_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./VEGFR1-Axitinib/4AGC_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./VEGFR1-Axitinib/4AG8_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./VEGFR1-Axitinib/4AGC_fixed_ph7.4.pdb.apo/trajectory.dcd
- **./ALK-Crizotinib/2XP2_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./ALK-Crizotinib/2YFX_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./ALK-Crizotinib/4ANQ_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./ALK-Crizotinib/4ANS_fixed_ph7.4.pdb.ligand/trajectory.dcd
- **./ALK-Crizotinib/2XP2_fixed_ph7.4.pdb.apo/trajectory.dcd**
- ./ALK-Crizotinib/2YFX_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./ALK-Crizotinib/4ANQ_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./ALK-Crizotinib/4ANS_fixed_ph7.4.pdb.apo/trajectory.dcd
- **./MET-Crizotinib/2WGJ_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./MET-Crizotinib/2WGJ_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BRAF-Dabrafenib/4XV2_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BRAF-Dabrafenib/5CSW_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BRAF-Dabrafenib/5HIE_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BRAF-Dabrafenib/4XV2_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BRAF-Dabrafenib/5CSW_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BRAF-Dabrafenib/5HIE_fixed_ph7.4.pdb.apo/trajectory.dcd
- **./BCRABL-Dasatinib/2GQG_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./BCRABL-Dasatinib/4XEY_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BCRABL-Dasatinib/2GQG_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BCRABL-Dasatinib/4XEY_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BCRABL-Imatinib/2HYY_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BCRABL-Imatinib/3PYY_fixed_ph7.4.pdb.ligand/trajectory.dcd
- ./BCRABL-Imatinib/2HYY_fixed_ph7.4.pdb.apo/trajectory.dcd
- ./BCRABL-Imatinib/3PYY_fixed_ph7.4.pdb.apo/trajectory.dcd
- **./VEGFR1-Regorafenib/2QU5_fixed_ph7.4.pdb.ligand/trajectory.dcd**
- ./VEGFR1-Regorafenib/2QU5_fixed_ph7.4.pdb.apo/trajectory.dcd

**Comment by jchodera** [comment]
Are these files in the repo, or on the cluster? If on the cluster, what's the path?

**Comment by bas-rustenburg** [comment]
/cbio/jclab/home/rustenburg/constph-grant/systems/simulations

**Comment by jchodera** [comment]
Is there any other output besides the contents of the `failed.txt` file, which is simply
```
Particle position is NaN
```
?

**Comment by bas-rustenburg** [comment]
Yes, but somehow all those log files have ended up in my home directory, instead of the qsub directory. They're called by the PDB file names, plus the usual `.o123456` jobname output.

**Comment by bas-rustenburg** [comment]
Preliminary update on this: a decent amount of pdb files seem to have botched up loops, which may be the source for some of the simulation crashes that happened immediately or on a short time scale.

**Comment by jchodera** [comment]
We are looking into more robust loop building alternatives for these
structures. At the very least, we will need to see how robust the MCCE
calculations are to the loop modeling strategy on a few specific cases.
